src/commands/addBinding/BindingCreateStep.ts

```diff
@@ -27,7 +27,9 @@ export class BindingCreateStep extends AzureWizardExecuteStep<IBindingWizardCont
         binding.direction = bindingTemplate.direction;
 
         for (const b of bindingTemplate.settings) {
-            binding[b.name] = getBindingSetting(context, b);
+            if (getBindingSetting(context, b) !== "undefined") {
```

Comment:
I'd store the result from `getBindingSetting(...)` to avoid needing to call it twice. e.g.:

```typescript
const bindingSetting = getBindingSetting(context, b);
if (bindingSetting !== "undefined") {
    binding[b.name] = bindingSetting;
}
```